### PR TITLE
mark processor down for some time not updating resolve ts

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
review after https://github.com/pingcap/ticdc/pull/154

mark processor down for some time not updating resolve ts

### What is changed and how it works?
mark processor down for some time not updating resolve ts.
and clear up data dispatch tables to other captures laster.

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### CODE REVIEW

File: cdc/owner.go [hunk 1]
```diff
@@ -384,7 +430,26 @@ func (o *ownerImpl) loadChangeFeedInfos(ctx context.Context) error {
...
 
 		if cfInfo, exist = o.changeFeedInfos[changeFeedID]; exist {
+			for cid, pinfo := range etcdChangeFeedInfo {
+				if _, ok := cfInfo.processorLastUpdateTime[cid]; !ok {
+					cfInfo.processorLastUpdateTime[cid] = time.Now()
+					continue
+				}
+
+				oldPinfo, ok := cfInfo.ProcessorInfos[cid]
+				if !ok || oldPinfo.ResolvedTs != pinfo.ResolvedTs {
```
Comment (amyangfei) [inline cdc/owner.go]: both `ResolvedTs` and `CheckpointTs` change can trigger the update of `last update time`?

Comment (july2993) [reply cdc/owner.go]: only check ResolvedTS here do you means check it both?

File: cdc/owner.go [hunk 2]
```diff
@@ -34,6 +35,8 @@ import (
 	"go.uber.org/zap"
 )
 
+var markProcessorDownTime = time.Minute
```
Comment (amyangfei) [inline cdc/owner.go]: It seems the interval that processor updates `resolvedTs` or `checkpointTs` is not always less than 1 minute. Besides shall we allow several times update failure of processor and do we need to increase this value?

File: cdc/owner.go (hunk 1 above)
Comment (amyangfei) [reply cdc/owner.go]: `if !ok || oldPinfo.ResolvedTs != pinfo.ResolvedTs || oldPinfo.CheckpointTs != pinfo.CheckpointTs`
like this?

**Pull request closed without merging**